proto/greenfield/storage/tx.proto

```diff
@@ -679,3 +681,24 @@ message MsgCancelUpdateObjectContent {
 }
 
 message MsgCancelUpdateObjectContentResponse {}
+
+message MsgSetBucketFlowRateLimit {
+  option (cosmos.msg.v1.signer) = "operator";
+
+  // operator defines the account address of the operator, either the object owner or the updater with granted permission.
+  string operator = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // bucket_name defines the name of the bucket
+  string bucket_name = 2;
+  // bucket_owner defines the account address of the bucket owner
+  string bucket_owner = 3 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
`payment_address defines an account address specified by bucket owner` - is this wrong?

Reply:
fixed